x-pack/plugins/spaces/public/nav_control/components/spaces_menu.tsx

```diff
@@ -85,8 +85,11 @@ class SpacesMenuUI extends Component<Props> {
           };
 
     return (
-      <>
+      <Fragment>
         <EuiSelectable
+          aria-label={i18n.translate('xpack.spaces.navControl.spacesMenu.spacesAriaLabel', {
```

Comment:
Question: Is this when closing the Spaces selector on the top nav?

I found this, but I'm not sure if this is the right spot:
<img width="629" alt="Screenshot 2023-03-29 at 11 22 11 AM" src="https://user-images.githubusercontent.com/21210601/228587902-782833b5-a45a-4d80-b5c8-815e62396fa7.png">

Reply:
Yes it's for the spaces navigation popover in the main menu/header of Kibana

Reply:
<img width="598" alt="Screenshot 2023-03-30 at 10 35 01 AM" src="https://user-images.githubusercontent.com/21210601/228870903-d82764ef-704b-47e5-9ab5-6e26eb780b18.png">
 
Ahhh got it!